packages/react/src/components/UIShell/HeaderMenuButton.js

```diff
@@ -38,8 +38,8 @@ const HeaderMenuButton = ({
     'aria-label': ariaLabel,
     'aria-labelledby': ariaLabelledBy,
   };
-  const menuIcon = renderMenuIcon ? renderMenuIcon : <Menu20 />;
-  const closeIcon = renderCloseIcon ? renderCloseIcon : <Close20 />;
+  const menuIcon = renderMenuIcon ? renderMenuIcon : <Menu size={20} />;
+  const closeIcon = renderCloseIcon ? renderCloseIcon : <Close size={20} />;
```

Comment:
```suggestion
  const menuIcon = renderMenuIcon || <Menu size={20} />;
  const closeIcon = renderCloseIcon || <Close size={20} />;
```

a simpler alternative maybe